wagtail/images/wagtail_hooks.py

```diff
@@ -80,9 +84,30 @@ def register_image_feature(features):
 
     # define how to convert between editorhtml's representation of images and
     # the database representation
-    features.register_converter_rule('editorhtml', 'image', [
-        EmbedTypeRule('image', ImageEmbedHandler)
-    ])
+    features.register_converter_rule('editorhtml', 'image', EditorHTMLImageConversionRule)
+
+    # define a draftail plugin to use when the 'image' feature is active
+    features.register_editor_plugin(
+        'draftail', 'image', draftail_features.EntityFeature({
+            'type': ENTITY_TYPES.IMAGE,
+            'icon': 'image',
+            'source': 'ImageSource',
+            'block': 'ImageBlock',
+            # We do not want users to be able to copy-paste hotlinked images into rich text.
+            # Keep only the attributes Wagtail needs.
+            'attributes': ['id', 'src', 'alt', 'format'],
+            # Keep only images which are from Wagtail.
+            'whitelist': {
+                # TODO Would be nice to simply filter by "id is present" instead.
+                # 'id': '\d*',
+                'src': '^/',
```

Comment:
Is it safe to assume all image paths start with `/` in Wagtail? My gut feeling is "no not at all" (if the images come from say S3), but worth checking.